src/index.html

```diff
@@ -3,11 +3,330 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link
+      rel="shortcut icon"
+      type="image/vnd.microsoft.icon"
+      href="//static.bose.com/etc/designs/bose/consumer-products-2016/favicon.ico"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
+    <header class="header">
+      <div class="header__content-wraper">
+        <article class="header__control-wraper">
+          <a
+          href="#"
+          class="header__logo"
+          >
+            <img
+              src="/images/bose-logo.png"
+              alt="Bose logo"
+              class="header__logo-image"
+            >
+          </a>
+          <div class="header__icons">
+            <a
+              href="tel: +1234555-55-55"
+              class="header__phone-link"
+            >
+            </a>
+
+            <button class="header__menu-toggler"></button>
+          </div>
+        </article>
+
+        <article class="menu header__menu" id="menuOverlay">
+          <div class="menu__wraper">
+            <div class="menu__controls">
+              <a
+                href="#"
+                class="header__logo header__logo--menu"
+              >
+                <img
+                  src="/images/bose-logo.png"
+                  alt="Bose logo"
+                  class="header__logo-image"
+                >
+              </a>
+
+              <button  class="menu__hide"></button>
+            </div>
+
+            <nav class="nav">
+              <ul class="nav__list">
```

Comment:
would be cool if links actually get user to the relevant sections on the site

Reply:
Have added ids thank you